### ISSUE TITLE
Application crash

### ISSUE THREAD

**Issue by kr0000** [quikkly/react-native#1]
After install library and start application:

dyld: Symbol not found: __T0BOWV
  Referenced from: /Users/X/Library/Developer/CoreSimulator/Devices/F33DEB7F-0F6E-40F5-8331-29B4ACC87F30/data/Containers/Bundle/Application/09F33A56-39B7-40AF-9DA6-2020CB0C4C65/X/Frameworks/Quikkly.framework/Quikkly
  Expected in: /Users/X/Library/Developer/CoreSimulator/Devices/F33DEB7F-0F6E-40F5-8331-29B4ACC87F30/data/Containers/Bundle/Application/09F33A56-39B7-40AF-9DA6-2020CB0C4C65/X/Frameworks/libswiftCore.dylib
 in /Users/X/Library/Developer/CoreSimulator/Devices/F33DEB7F-0F6E-40F5-8331-29B4ACC87F30/data/Containers/Bundle/Application/09F33A56-39B7-40AF-9DA6-2020CB0C4C65/X/Frameworks/Quikkly.framework/Quikkly
(lldb)

**Comment by janekp** [comment]
The RN plugin, iOS SDK and Android SDK are separate components. 

The support for Xcode 10 / Swift 4.2 was added in 3.4.7. However, the updated RN plugin hasn't been published to NPM yet, because of a separate Android issue (in order to keep version numbers in sync).

Please try to use a newer pod meanwhile:

`pod 'Quikkly', :git => 'https://github.com/quikkly/ios-sdk.git', :tag => "3.4.7"`

**Comment by kr0000** [comment]
Now it's better, please let me know if You release stable version also for Android. Thanks.

**Comment by janekp** [comment]
Yes. There's a problem with one Samsung model. RN 3.4.7 will be published after that.
